src/combinatorics/simple_graph/clique.lean

```diff
@@ -68,4 +80,91 @@ variables {G H}
 lemma is_n_clique.mono (h : G ≤ H) : G.is_n_clique n s → H.is_n_clique n s :=
 by { simp_rw is_n_clique_iff, exact and.imp_left (is_clique.mono h) }
 
+@[simp] lemma is_n_clique_bot_iff : (⊥ : simple_graph α).is_n_clique n s ↔ n ≤ 1 ∧ s.card = n :=
+begin
+  rw [is_n_clique_iff, is_clique_bot_iff],
+  refine and_congr_left _,
+  rintro rfl,
+  exact card_le_one.symm,
+end
+
+end n_clique
+
+/-! ### Graphs without cliques -/
+
+section clique_free
+variables {n : ℕ}
+
+/-- `G.clique_free n` means that `G` has no `n`-cliques. -/
+def clique_free (n : ℕ) : Prop := ∀ t, ¬ G.is_n_clique n t
+
+variables {G H}
+
+lemma clique_free_bot (h : 2 ≤ n) : (⊥ : simple_graph α).clique_free n :=
+begin
+  rintro t ht,
+  rw is_n_clique_bot_iff at ht,
+  linarith,
+end
+
+lemma clique_free.mono (h : G ≤ H) : H.clique_free n → G.clique_free n :=
```

Comment:
Just to check: do we use `mono` for antitone functions too?

Reply:
We're being inconsistent about this. My rule of thumb is to use always use `mono` unless the function is binary and monotone in one argument and antitone in the other, in which case I use `mono`/`anti` rather than `mono_left`/`mono_right`.

Reply:
We're in that situation here, right? If I'm not mistaken, if `n <= m`, then `G.clique_free n -> G.clique_free m`.

Reply:
Oh shoot, we are